### DISCUSSION TITLE
limited details for Cancelations / Void / Deny - Fetch Appointment Details Webhook:

### DISCUSSION THREAD

**Discussion by OnWednesdaysWeWearFlannel** [freightapis/ssc#11] [Q&A]
Hello Team, 
When implementing the Fetch Appointment Details web hook event the entity is un-associated from the load and therefore the details aren't available. So we are only going to send back. Let us know if we need to implement the additional details in the future for voids / rejections/cancellations and deny. 

{
    "identifiers": 
{
    "primaryReferenceNumber": "LOAD1234",
    "stopSequenceNumber": 1
},
"currentAppointment": 
{
        "appointmentStatus": "CANCELED"
    },
    "occurredAt": "2023-04-15T08:02:23.123Z"
}

Thank you, 
Shipper implementation

**Comment by miller79** [comment]
I may not be understanding your issue specifically, but are you specifically talking about the [Fetch Available Appointments Event](https://freightapis.github.io/ssc/#tag/Webhook-Events/operation/fetch-available-appointments-event) webhook? This webhook is intended to be tied to the `requestIdentifier` which is provided as a return value of the [Fetch Available Appointments](https://freightapis.github.io/ssc/#tag/Appointments/operation/fetch-available-appointments) endpoint when a `PENDING` status is returned. I don't know what you mean when you say the entity is un-associated, but if you could provide some more details and the specific use case you are trying to solve, I can offer some additional information for you.
